src/main/java/chaterpillar/Chaterpillar.java

```diff
@@ -0,0 +1,77 @@
+package chaterpillar;
```

Comment:
Overall, I like that your code is easily understandable and readable.
Good coding standard!